app/controllers/api/account.php

```diff
@@ -524,7 +525,40 @@
                         'registration' => \time(),
                         'reset' => false,
                         'name' => $name,
+                        'prefs' => [],
                     ], ['email' => $email]);
+
+                    $createUserEvent = clone $audits;
+
+                    $createUserEvent
+                        ->setParam('eventData', $response->output($user, Response::MODEL_USER))
```

Comment:
Since this can be quite the heavy task with all the things going on - we might want to avoid calling `$response->output` three time to spare some compute 👍🏻